### PR TITLE
Suhan

### PR DESCRIPTION
**Pull request by jsh6269**
pull request

### CODE REVIEW

File: READMESuhan.md [hunk 1]
```diff
@@ -1 +1,18 @@
-장수한입니다.
\ No newline at end of file
+# Suhan  
+Hi, I'm suhan. I'm currently studying computer engineering.  
+<br>
+
```
Comment (rokhlee) [inline READMESuhan.md:4]: nice to meet you!

File: READMESuhan.md [hunk 2]
```diff
@@ -1 +1,3 @@
-장수한입니다.
\ No newline at end of file
+| 이름 | 학과 | 학번 |
```
Comment (minss7) [inline READMESuhan.md:16]: good

File: READMESuhan.md (hunk 1 above)
Comment (jsh6269) [reply READMESuhan.md:4]: nice to meet you, too!

File: READMESuhan.md [hunk 3]
```diff
@@ -1 +1,3 @@
...
+| :---: | :---: | :---: |
```
Comment (minss7) [inline READMESuhan.md:17]: hi

File: READMESuhan.md (hunk 2 above)
Comment (minss7) [inline READMESuhan.md:16]: hii

**Pull request merged**